microscope/reporter_tokenizer.py

```diff
@@ -7,12 +7,12 @@
 import re
 from typing import List
 
-from reporters_db import EDITIONS, VARIATIONS_ONLY
+from microscope.helpers import REPORTER_STRINGS
 
 # We need to build a REGEX that has all the variations and the reporters in
 # order from longest to shortest.
-REGEX_LIST = list(EDITIONS.keys()) + list(VARIATIONS_ONLY.keys())
-REGEX_LIST.sort(key=len, reverse=True)
+
+REGEX_LIST = sorted(REPORTER_STRINGS, key=len, reverse=True)
```

Comment:
Hm, two thoughts here. Now that REPORTER_STRINGS is a `Set`, does sorting work properly? And if so (thanks py3?) should we rename REGEX_LIST?

Reply:
That's why we're using `sorted()` instead of `.sort()` -- that line copies the sorted-by-insertion-order set into a sorted-by-length list.

Reply:
Ah ha. Missed that. Great.